### PR TITLE
automated: linux: Add ostree test

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@foundries.io>

### CODE REVIEW

File: automated/linux/ostree/ostree.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2021 Foundries.io Ltd.
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+OSTREE_HASH=""
+
```
Comment (roxell) [inline automated/linux/ostree/ostree.sh:11]: Can we set SKIP_INSTALL to false so its clear.

File: automated/linux/ostree/ostree.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
```
Comment (roxell) [inline automated/linux/ostree/ostree.sh:1]: This file mixing tab and spaces please fix.

File: automated/linux/ostree/ostree.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    scope:
+        - functional
+
+    devices:
+        - imx8mm
+        - bcm2711-rpi-4-b
+
+params:
+        OSTREE_HASH: ""
+
```
Comment (roxell) [inline automated/linux/ostree/ostree.yaml:26]: Add SKIP_INSTALL and pass it in './ostree.sh -m "${OSTREE_HASH}" -s "${SKIP_INSTALL}"'

File: automated/linux/ostree/ostree.sh (hunk 1 above)
Comment (mwasilew) [reply automated/linux/ostree/ostree.sh:12]: Done

File: automated/linux/ostree/ostree.yaml (hunk 3 above)
Comment (mwasilew) [reply automated/linux/ostree/ostree.yaml:27]: Done

**Pull request merged**